docker-compose.yml

```diff
@@ -1,6 +1,12 @@
+#BASE_URI_ARG argument is passed to dockerfile from command line to rebase the ui path For eg: BASE_URI_AR=/my-ui/. 
+#If passed empty, then / is appended. The Uri must begin and end with / 
 version: '3.5'
 services:
   dcsa-ebl-ui:
-    build: .
+    build: 
+      context: .
+      args: 
+        - BASE_URI_ARG=/edoc-ui/
```

Comment:
The default for local testing should be `/`

Reply:
done and pushed the changes